bridge/engine/EngineMixer.cpp

```diff
@@ -3528,6 +3549,33 @@ void EngineMixer::processRecordingMissingPackets(const uint64_t timestamp)
     }
 }
 
+bool EngineMixer::needToUpdateMinUplinkEstimate(const uint32_t curEstimate, const uint32_t oldEstimate) const
+{
+    // For screensharing (a.k.a. 'slides') we have a minumum allowed bitrate of 900 kbps,
+    // so it does not worth to react on the fluctuation below ~10% of this value.
+    return abs((int64_t)oldEstimate - (int64_t)curEstimate) > 100;
+}
+
+uint32_t EngineMixer::getBarbellsMinUplinkBitrate() const
```

Comment:
perhaps 
getMinRemoteClientDownlinkBandwidth 
though we get it by iterating the barbells recording of such value.

Reply:
renamed